src/mmw/mmw/settings/gwlfe_settings.py

```diff
@@ -0,0 +1,394 @@
+    'n146': 0,  # Pathogen Loads (Wastewater Treatment Plants): Future (orgs/month) # NOQA
+    'n147': 0,  # Pathogen Loads (Septic Systems): Future (orgs/month)
+    'n148': 0,  # Pathogen Loads (Urban Areas): Future (orgs/month)
+    'n149': 0,  # Pathogen Loads (Wildlife): Future (orgs/month)
+    'n150': 0,  # Pathogen Loads (Total): Future (orgs/month)
+    'n151': 0,  # Pathogen Loads: Percent Reduction (%)
+    'InitNgN': 2373,  # Initial Non-Grazing Animal Totals: Nitrogen (kg/yr)
+    'InitNgP': 785,  # Initial Non-Grazing Animal Totals: Phosphorus (kg/yr)
+    'InitNgFC': 3.38e+9,  # Initial Non-Grazing Animal Totals: Fecal Coliforms (orgs/yr) # NOQA
+    'NGAppSum': 0.55,  # Non-Grazing Manure Data Check: Land Applied (%)
+    'NGBarnSum': 0.28,  # Non-Grazing Manure Data Check: In Confined Areas (%)
+    'NGTotSum': 0.83,  # Non-Grazing Manure Data Check: Total (<= 1)
+    'InitGrN': 2373,  # Initial Grazing Animal Totals: Nitrogen (kg/yr)
+    'InitGrP': 785,  # Initial Grazing Animal Totals: Phosphorus (kg/yr)
+    'InitGrFC': 3.38e+9,  # Initial Grazing Animal Totals: Fecal Coliforms (orgs/yr) # NOQA
+    'GRAppSum': 0.52,  # Grazing Manure Data Check: Land Applied (%)
+    'GRBarnSum': 0.18,  # Grazing Manure Data Check: In Confined Areas (%)
+    'GRTotSum': 1,  # Grazing Manure Data Check: Total (<= 1)
+    'AnimalFlag': 1,  # Flag: Animal Layer Detected (0 No; 1 Yes)
+    'WildOrgsDay': 5.0e+8,  # Wildlife Loading Rate (org/animal/per day)
+    'WildDensity': 25,  # Wildlife Density (animals/square mile)
+    'WuDieoff': 0.9,  # Wildlife/Urban Die-Off Rate
+    'UrbEMC': 9600,  # Urban EMC (org/100ml)
+    'SepticOrgsDay': 2.0e+9,  # Septic Loading Rate (org/person per day)
+    'SepticFailure': 0,  # Malfunctioning System Rate (0 - 1)
+    'WWTPConc': 200,  # Wastewater Treatment Plants Loading Rate (cfu/100ml)
+    'InstreamDieoff': 0.5,  # In-Stream Die-Off Rate
+    'AWMSGrPct': 0,  # Animal Waste Management Systems: Livestock (%)
+    'AWMSNgPct': 0,  # Animal Waste Management Systems: Poultry (%)
+    'RunContPct': 0,  # Runoff Control (%)
+    'PhytasePct': 0,  # Phytase in Feed (%),
+    'AnimalName': ['Dairy Cows', 'Beef Cows', 'Broilers', 'Layers',
+                   'Hogs/Swine', 'Sheep', 'Horses', 'Turkeys', 'Other'],
+    'NumAnimals': np.zeros(9),
+    'GrazingAnimal': ['Y', 'Y', 'N', 'N',
+                      'N', 'Y', 'Y', 'N', 'N'],
+    'AvgAnimalWt': np.array([640, 360, 0.9, 1.8,
+                             61, 50, 500, 6.8, 0]),  # Average Animal Weight (kg) # NOQA
+    'AnimalDailyN': np.array([0.44, 0.31, 1.07, 0.85,
+                              0.48, 0.37, 0.28, 0.59, 0]),  # Animal Daily Loads: Nitrogen (kg/AEU) # NOQA
+    'AnimalDailyP': np.array([0.07, 0.09, 0.30, 0.29,
+                              0.15, 0.10, 0.28, 0.59, 0]),  # Animal Daily Loads: Phosphorus (kg/AEU) # NOQA
+    'FCOrgsPerDay': np.array([1.0e+11, 1.0e+11, 1.4e+8, 1.4e+8,
```

Comment:
This set of variables (`AnimalName` - `FCOrgsPerDay`) is a little hard to keep track of and validate, since each array contains one slice of data, linked by index. While the model does expect the values in this format, I will look into refactoring this into a structure that keeps linked data together.